pkg/scheduler/framework/plugins/nodeports/node_ports_test.go

```diff
@@ -170,7 +171,7 @@ func TestPreFilterDisabled(t *testing.T) {
 	p, _ := New(nil, nil)
 	cycleState := framework.NewCycleState()
 	gotStatus := p.(framework.FilterPlugin).Filter(context.Background(), cycleState, pod, nodeInfo)
-	wantStatus := framework.NewStatus(framework.Error, `error reading "PreFilterNodePorts" from cycleState: not found`)
+	wantStatus := framework.AsStatus(fmt.Errorf(`error reading "PreFilterNodePorts" from cycleState: %w`, fmt.Errorf("not found")))
```

Comment:
remove word error

Reply:
Thank you very much for your review @alculquicondor
All the Suggestions have been applied.